### ISSUE TITLE
Apollo server unit tests

### ISSUE THREAD

**Issue by sparrowDom** [OriginProtocol/origin-js#576]
Create unit/integration tests for Apollo server

**Comment by franckc** [comment]
Super. Thanks for filing this. We should also add unit tests to the EventListener.

**Comment by sparrowDom** [comment]
yeah my bad... here it is: https://github.com/OriginProtocol/origin-js/issues/577
